spec.bs

```diff
@@ -3608,6 +3609,28 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
 
 </div>
 
+# Feature Detection # {#feature-detection}
+
+The {{ProtectedAudience/queryFeatureSupport()}} method permits checking what functionality is available in the current implementation, in order to help deploy new features.  The return values specified in this specifications are for an implementation that fully implements it.

```

Comment:
```suggestion
The {{ProtectedAudience/queryFeatureSupport()}} method permits checking what functionality is available in the current implementation, in order to help deploy new features.  The return values specified here are for an implementation that fully implements it.

```

Reply:
What's the trailing 'it' in the revised sentence bound to..?

Reply:
I'm not sure how my edit changes that, but maybe I misunderstood before? Just a sugggestion, feel free to ignore it (but there is a typo in `specifications`, right?)

Reply:
Well, "it" used to bind to the "specifications"[sic --- you are right that it should be singular], but with 'here' there is nothing for it to bind to since 'here' is an adverb and not a noun?

Reply:
One day I will learn English. Maybe...